uitools/examples/cpp_quick/src/demos/ScalebarDemo.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
```

Comment:
(this is the _"As above"_ I was referring to)
❓ shouldn't we use the Apache license?

```
/*******************************************************************************
 *  Copyright 2012-2022 Esri
 *
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions and
 *  limitations under the License.
 ******************************************************************************/
 ```

Reply:
Yes, you are correct. I guess my thought process was to keep consistent for now and go through and change them all to Apache as a separate task.

Reply:
Personally, I prefer the Starfish Principle over consistency: if something is wrong/broken, let's fix it here and now, so we don't need to come back to it later. 
It's up to you, but I would encourage using the correct copyright notice going forward; and go through and change the rest later when we'll make time for it.